packages/types/src/interfaces.ts

```diff
@@ -8,6 +8,7 @@ import {IEvent} from '@polkadot/types/types';
 
 export interface Entity {
   id: string;
+  name: string;
```

Comment:
I don't think this is practical to add a name to all entities. This will add another column to every db